### PR TITLE
Vuefy color input form element

### PR DESCRIPTION
**Pull request by OlegZharkov**
This PR implements Vue version of input form color picker

xref: https://github.com/galaxyproject/galaxy/issues/11076#issuecomment-958261281
The old interface:
image

The new interface:
colorfin

## How to test the changes?
- [x] Instructions for manual testing are as follows:
  1. Run with debug tools and choose `color_param`
  2. Colors should be there

## License
- [x] I agree to license these contributions under [Galaxy's current license](https://github.com/galaxyproject/galaxy/blob/dev/LICENSE.txt).
- [x] I agree to allow the Galaxy committers to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT). If this condition is an issue, uncheck and just let us know why with an e-mail to galaxy-committers@lists.galaxyproject.org.

### CODE REVIEW

**Comment by hexylena** [comment]
Ahh this will make people happier. I'll have to update the training material since we mentioned in a couple places that it was an intentionally restricted colour palette to prevent people choosing bad colours.

**Comment by guerler** [comment]
Yes this is very nice. Thanks @OlegZharkov. Previously the idea was that colors can be re-picked easily since a restricted palette is shown, similar to palettes in Excel, although the backend had no restrictions in accepting any color.

File: client/src/components/Form/Elements/FormColor.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <label class="pt-1 cursor-pointer" :for="id">Select a color</label>
+        </b-col>
+    </b-row>
+</template>
+
+<script>
+export default {
+    props: {
+        value: {
+            type: String,
```
Comment (guerler) [inline client/src/components/Form/Elements/FormColor.vue:16]: Value can be `null`, so removing the type attribute here resolves this.

**Comment by OlegZharkov** [comment]
thanks @guerler!

**Pull request merged**